Makefile

```diff
@@ -662,6 +666,26 @@ node_modules: package-lock.json
 	npm install --no-save
 	@touch node_modules
 
+.PHONY: npm-cache
+npm-cache: .npm-cache $(FOMANTIC_WORK_DIR)/node_modules/fomantic-ui
+
+.npm-cache: package-lock.json
+	rm -rf .npm-cache
+	$(eval ESBUILD_VERSION := `node -p "require('./package-lock.json').dependencies.esbuild.version"`)
+	$(eval ESBUILD_PKGS := esbuild-{darwin-64,linux-{arm,arm64,32,64},windows-{32,64}}@$(ESBUILD_VERSION))
+	npm config --userconfig=.npmrc set cache=.npm-cache
+	rm -rf node_modules && npm install --no-save
+	npm config --userconfig=$(FOMANTIC_WORK_DIR)/.npmrc set cache=../../.npm-cache
+	echo $(ESBUILD_PKGS) fsevents@1 fsevents@2 | tr " " "\n" | xargs -n 1 -P 4 npm cache add
```

Comment:
I think these `fsevents` can be dropped, they are generally considered optional dependencies, so nothing should break if they are not there.

Reply:
Actually, I wonder where they are required in first place. I know some webpack 4 modules did need them but if I run a `npm ls fsevents` on a Mac on master, I get an empty result. So maybe they are already obsolete?

Reply:
Okay those are actually for `gulp`, which requires `chokidar@2` and `fsevents@1`. `fsevents@2` isn't needed by anything as of now. My bad. PR is on the way.